src/main/scala/io/mesosphere/mesosaurus/Logging.scala

```diff
@@ -3,5 +3,5 @@ package io.mesosphere.mesosaurus
 import org.slf4j.{ Logger, LoggerFactory }
 
 trait Logging {
-  protected lazy val log = LoggerFactory.getLogger(getClass.getName)
+    protected lazy val log = LoggerFactory.getLogger(getClass.getName)
```

Comment:
Actually, manually doing formatting fixes is a huge waste of time.  Let's add [scalariform](https://github.com/sbt/sbt-scalariform) to the build in a separate patch.